lib/vagrant-openshift/action/install_origin_base_dependencies.rb

```diff
@@ -215,6 +215,12 @@ def call(env)
   sudo lvcreate -n docker-data -l 90%FREE /dev/${VG}
   sudo lvcreate -n docker-metadata -l 50%FREE /dev/${VG}
   sudo sed -i "s,^DOCKER_STORAGE_OPTIONS=.*,DOCKER_STORAGE_OPTIONS='-s devicemapper --storage-opt dm.datadev=/dev/${VG}/docker-data --storage-opt dm.metadatadev=/dev/${VG}/docker-metadata'," /etc/sysconfig/docker-storage
+
+  sudo lvcreate -n openshift-xfs-tmp -l 100%FREE /dev/${VG}
+  sudo mkfs.xfs /dev/${VG}/openshift-xfs-tmp
+  sudo mkdir -p /tmp/openshift
```

Comment:
At this point we'll be getting into territory I don't quite understand, I know the extended tests appear to run in /tmp/openshift so my plan was to push this onto the new xfs LV entirely, the change shouldn't affect any other test and will let this new test pass.

I assume you're thinking my new tests somehow re-initialize all the test infra in another location and initialize a master/node from there? I will need to do some digging to find out how to do this.